pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -197,9 +213,25 @@ func (n *NetworkPolicyController) syncInternalGroup(key string) error {
 		klog.Errorf("Failed to update ClusterGroup %s GroupMembersComputed condition to %s: %v", cg.Name, v1.ConditionTrue, err)
 		return err
 	}
+	n.triggerParentGroupSync(grp)
 	return n.triggerCNPUpdates(cg)
 }
 
+func (n *NetworkPolicyController) triggerParentGroupSync(grp *antreatypes.Group) {
+	// TODO: if the group nest level increases, this will no longer be a valid condition
+	//  to check whether a group can possibly have parents.
```

Comment:
I do not understand this comment. Probably you should clarify more here.

Reply:
Done